app/konnect/getting-started/index.md

```diff
@@ -40,6 +40,48 @@ New to {{site.konnect_saas}}? Get started with the basics through the web app:
 
     [Implement the service &gt;](/konnect/getting-started/implement-service)
 
+
+4. **Publish the service to Dev Portal**
+
+    Set up a sample API spec and description for the {{site.konnect_short_name}}
+    service and publish the service to the Dev Portal.
+
+    [Publish the service &gt;](/konnect/getting-started/dev-portal)
+
+## Get started with an API spec
+
+Have an API spec and want to share your API with developers? Get started by
```

Comment:
I asked this question earlier and didn't get a clear yes/no, mostly because there's no integration with Insomnia at the moment. I don't see why we wouldn't recommend using it separately, though. Either way, they're using _some_ tool to write the spec.